web/docs/general/wasp-ts-config.md

```diff
@@ -4,9 +4,9 @@ title: Wasp TypeScript config (*.wasp.ts)
 
 import DiscordLink from '@site/blog/components/DiscordLink';
 
-:::caution Requires Wasp >= 0.15
-This document assumes your app works with Wasp >= 0.15.  
-If you haven't migrated your app yet, follow the [migration instructions](../migration-guides/migrate-from-0-14-to-0-15.md) and verify everything works. After that, come back here and try out the new Wasp TS config.
+:::caution Requires Wasp >= 0.16.3
```

Comment:
I bumped this up since we made a change in the public API with 0.16.3 (`HttpRoute` by @FranjoMindek).